.github/workflows/deploy-gh-pages.yml

```diff
@@ -0,0 +1,22 @@
+name: Deploy To Github Pages
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2
+      - name: Installs and builds package
+        run: |
+          npm install
+          npm run build
```

Comment:
I think we'll have to run 

```
# ./node_modules/.bin/typings install (for Typescript definition files)
Generate endpoint definition from stone spec:

cd codegen
./run_codegen.sh
```

as well from https://github.com/dropbox/dropbox-api-v2-explorer/blob/master/ReadMe.md?

Reply:
Ill need to update the readme.  forgot to do that with the update.  - typescript doe not use typings anymore so the first comment is no longer needed.

Looking into last comment